src/App.js

```diff
@@ -18,7 +19,7 @@ const tabs = [
 ];
 
 const App = () => (
-  <h1>React tabs</h1>
+  <Tabs tabs={tabs} />
```

Comment:
In this task you're supposed to render selected tab content in the App component